src/views/templates/template-details/TemplateDetailsView.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from 'react';
+import csx from './TemplateDetailsView.scss';
+import { Button } from 'shared/ui';
+
+import EditIcon from '@material-ui/icons/Edit';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import MenuBookIcon from '@material-ui/icons/MenuBook';
+import ShareIcon from '@material-ui/icons/Share';
+
+import { RouteChildrenProps } from 'react-router';
+
+interface TemplateDetailsViewProps extends RouteChildrenProps<{ id: string }> {}
+
+export const TemplateDetailsView = ({ match }: TemplateDetailsViewProps) => {
```

Comment:
This should be created as default export. All components with View postfix should be exported default. Check TemplatesView for reference